cpp/include/raft/spatial/knn/detail/knn_brute_force_faiss.cuh

```diff
@@ -22,23 +22,21 @@
 
```

Comment:
Maybe we should just rename this to `knn_brute_force.cuh`? We should also be able to move it to `raft/neighbors/detail`